packages/cli-lib/lib/uploadFolder.js

```diff
@@ -40,12 +41,14 @@ function getFilesByType(files) {
       cssAndJsFiles.push(file);
     } else if (extension === 'html') {
       templateFiles.push(file);
+    } else if (extension === 'json') {
```

Comment:
I suspect that we want to be more specific here and match only `theme.json`, `fields.json` that is not inside `*.module` and `serverless.json`.

Reply:
On second thought, maybe that is covered by how module files are already uploaded first. Might be good to add test coverage of this either way.

Reply:
Exactly, and I can't see a reason to treat `*.functions` files differently.

Adjusted test coverage to include json files